### PR TITLE
added java book in hindi

### PR DESCRIPTION
**Pull request by anishhh238**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Added Java prog. lang. book link in Hindi 

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-
+* [Java](#java)
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+### <a id="java"></a>Java
```
Comment (shrn01) [inline books/free-programming-books-hi.md:12]:
```suggestion
### Java
```

File: books/free-programming-books-hi.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 * [C++](#cpp)
-
+* [Java](#java)
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
```
Comment (shrn01) [inline books/free-programming-books-hi.md:11]:
```suggestion


```
Please add 2 spaces after each section

File: books/free-programming-books-hi.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
 ### Index
 
 * [C++](#cpp)
-
+* [Java](#java)
```
Comment (shrn01) [inline books/free-programming-books-hi.md:4]:
```suggestion
* [Java](#java)

```
2 spaces after each section

**Review by shrn01 — COMMENTED** [review]
Please read docs/CONTRIBUTING.md about how the entries have to be added and the spacings and update accordingly

**Comment by eshellman** [comment]
not a book

**Pull request closed without merging**